lib/jekyll/commands/new.rb

```diff
@@ -121,23 +121,23 @@ def scaffold_path
         # unless the user opts to generate a blank blog or skip 'bundle install'.
 
         def after_install(path, options = {})
-          Jekyll.logger.info "New jekyll site installed in #{path.cyan}."
-          Jekyll.logger.info "Bundle install skipped." if options["skip-bundle"]
-
           unless options["blank"] || options["skip-bundle"]
             bundle_install path
           end
+
+          Jekyll.logger.info "New jekyll site installed in #{path.cyan}."
+          Jekyll.logger.info "Bundle install skipped." if options["skip-bundle"]
         end
```

Comment:
This change here would warrant a change to the test file [here](https://github.com/jekyll/jekyll/blob/master/test/test_new_command.rb#L44-L48)